File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/NativeAdActivity.java
package com.applovin.enterprise.apps.demoapp.ads.max.nativead;

import android.content.Intent;

import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem;
import com.applovin.enterprise.apps.demoapp.ui.DemoMenuActivity;

public class NativeAdActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Manual API", new Intent( this, ManualNativeAdActivity.class ) ),
                new DemoMenuItem( "Manual Late Binding API", new Intent( this, ManualNativeLateBindingAdActivity.class ) ),
                new DemoMenuItem( "Recycler View Ad Placer", new Intent( this, RecyclerViewNativeAdActivity.class ) )
        };
        return result;
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/NativeAdActivity.kt
package com.applovin.enterprise.apps.demoapp.ads.max.nativead

import android.content.Intent
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
import com.applovin.enterprise.apps.demoapp.ui.DemoMenuActivity

class NativeAdActivity: DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
        DemoMenuItem("Manual API", Intent(this, ManualNativeAdActivity::class.java)),
        DemoMenuItem("Manual Late Binding API", Intent(this, ManualNativeLateBindingAdActivity::class.java)),
        DemoMenuItem("Recycler View Ad Placer", Intent(this, RecyclerViewNativeAdActivity::class.java))
    )
}
